frontend/src/assets/styles/_extra.scss

```diff
@@ -616,3 +616,7 @@ a[href="https://www.mapbox.com/map-feedback/"]
 .link:focus {
   outline: revert;
 }
+
+.scrollable-container::-webkit-scrollbar {
+  width: 0;
+}
```

Comment:
This effectively hides the scrollbar - users should be able to identify and interact with scrollable content easily.

![missing-scrollbar](https://github.com/hotosm/tasking-manager/assets/51614993/bd8cc68c-1f3c-4b1a-9abd-973008e56780)

Reply:
You mean i should remove this and keep the scroll bar visible?

Reply:
The scroll bar should be visible when the content overflows the container.

Reply:
Sure! I will do that.